### ISSUE TITLE
token not saving

### ISSUE THREAD

**Issue by Grantfamily** [Razviar/mtga-pro-tracker#450]
whenever I launch, the token is empty and has to be reinstalled.

I'm sorry I have no idea how to replicate.

**Comment by Razviar** [comment]
Hello! Is your app updated to the latest version?

**Comment by Grantfamily** [comment]
v 1.7.9
[cid:8e3b7eda-a33e-4cbb-a011-e7c9510dc587]

Kevin
grantfamily@live.com

________________________________

**Comment by Razviar** [comment]
Can you please take a screenshot of app interface when you launch it?

**Comment by Grantfamily** [comment]
I sent the screenshot in my l
[Image]


ast email

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by Grantfamily** [comment]
attached.

Kevin E Grant
grantfamily@live.com

________________________________

**Comment by Razviar** [comment]
Sorry, attached image was not displayed here. Please use something like dropbox or imgur.com

**Comment by Grantfamily** [comment]
I do not have either of those.  the file is attached to this email.

Kevin
grantfamily@live.com

**Comment by Grantfamily** [comment]
do you have any update on the bug?

Kevin E Grant
grantfamily@live.com


________________________________

**Comment by Razviar** [comment]
No, sorry, this is something exceptional and we can't find out why it's
happening to you or replicate the bug, sorry. There's no ETA on fix, since
we have no idea why it's happening. I'll ask ads agency again.

пн, 8 июл. 2019 г. в 13:03, Kevin E Grant <notifications@github.com>:
